src/nupic/types/Sdr.hpp

```diff
@@ -35,7 +35,7 @@ using namespace std;
 
 namespace nupic {
 
-typedef vector<Byte>          SDR_dense_t;
+typedef vector<UInt>          SDR_dense_t; //TODO add templated types for SDR<DenseElemT, SparseElemT, CoordElemT> + default SDR<Byte, UInt, UInt> 
```

Comment:
this is the breaking change. I think it'd be good to have this option.

Reply:
Are you saying that the SDR internal dense format should be UInt?   That is not a good idea.